src/agent/input-tester/src/tester.rs

```diff
@@ -420,11 +420,11 @@ impl Tester {
 }
 
 fn exception_from_throw(exception: &Exception) -> bool {
-    match exception.description {
+    matches!(
```

Comment:
I think this is more confusing that the match statement.

Reply:
I agree; when matching against several cases, I suggest using a full `match` expr.

Reply:
reverted this one.